README.md

```diff
@@ -1 +1,49 @@
-# This is a simple TODO service
+# TODO Service
+TODO Service provides a framework for building a microservice in go-lang. 
+
+## Source code editing
+Please install visual studio code and the go language. The version used can be seen in the go.mod file. Also consider installing "Rich Go language support for Visual Studio Code" by the go team at Google. It provides several nifty features
```

Comment:
I would prefer we dont opinianate on the IDE to use. I don't think we have anything specific to IDEs that has to do (no pun intended) with the service.